### ISSUE TITLE
integration test for all the parameter changes needed by EC

### ISSUE THREAD

**Issue by turadg** [Agoric/Agoric#6859]
## What is the Problem Being Solved?

EC will need these GUI controls:
- https://github.com/agoric/dapp-econ-gov/issues/31
- https://github.com/Agoric/dapp-econ-gov/issues/32
- https://github.com/Agoric/dapp-econ-gov/issues/33

Ensure they're available on chain before starting the UI work.

## Description of the Design

Tests like `gov-replace-committee.js` or `test-psm-integration`.

## Security Considerations
--
## Scaling Considerations
--
## Test Plan

--

**Comment by otoole-brendan** [comment]
#31 has been superseded by https://github.com/Agoric/agoric-sdk/issues/6944 as it was in wrong repo

**Comment by turadg** [comment]
Closing as all the governance params and actions now have test coverage and have been tested on chain.
